fem/eltrans.hpp

```diff
@@ -443,11 +443,23 @@ class IntegrationPointTransformation
 class FaceElementTransformations : public IsoparametricTransformation
 {
 private:
+
+   // Bitwise OR of ConfigMasks
    int mask;
 
    IntegrationPoint eip1, eip2;
 
 public:
+
+   enum ConfigMasks
+   {
+      HAVE_ELEM1 =  1, // Element on side 1 is configured
+      HAVE_ELEM2 =  2, // Element on side 2 is configured
+      HAVE_LOC1  =  4, // Point transformation for side 1 is configured
+      HAVE_LOC2  =  8, // Point transformation for side 2 is configured
+      HAVE_FACE  = 16  // Face transformation is configured
```

Comment:
Use `///` to make these comments also doxygen documentation.

Reply:
Of course!  I've got to develop that habit.

Reply:
Also, it may be good to mention these in the documentation of https://github.com/mfem/mfem/blob/cc3a4475d298df742de0a98d9c4586c3fd02ef82/mesh/mesh.hpp#L986-L987

Reply:
Yes, @najlkin suggested something similar.  I'll take care of it.